core/src/main/resources/hudson/diagnosis/MemoryUsageMonitor/index.jelly

```diff
@@ -61,7 +61,7 @@ THE SOFTWARE.
           </j:otherwise>
         </j:choose>
       </div>
-      <img src="graph?type=${type}&amp;width=500&amp;height=300" />
+      <img src="graph?type=${type}&amp;width=500&amp;height=300" alt="null" />
```

Comment:
Since this is the memory usage monitor graph, I think that the alt text should say that.  The word "null" won't help a disabled user understand what is being shown in the image.
```suggestion
      <img src="graph?type=${type}&amp;width=500&amp;height=300" alt="memory usage monitor graph" />
```

Reply:
Resolved in 904cb3e4ac582523f2db3d70c1404cad5f8facb7